src/dataframe/insertion.jl

```diff
@@ -297,7 +297,10 @@ function _append_or_prepend!(df1::DataFrame, df2::AbstractDataFrame; cols::Symbo
                 deleteat!(col, 1:length(col) - nrow1)
             end
         end
-        @error "Error adding value to column :$(_names(df1)[current_col])."
+        @error "Error adding value to column :$(_names(df1)[current_col]). " *
+               "Maybe you have forgotten to ask for column " *
+               "element type promotion, which can be done " *
+               "by passing `promote=true` keyword argument."
```

Comment:
I'd say just "`promote=true`." (as people can se that litterally), or "the `promote=true` keyword argument".

Reply:
I will add `the` and remove backticks then.